.github/workflows/suite-native_production_ci.yml

```diff
@@ -75,3 +76,31 @@ jobs:
           --auto-submit
           --message ${{ github.sha }}
         working-directory: suite-native/app
+  androidAPK:
```

Comment:
Why does the app have to be build separately? Could not we use the already built `.apk` from `android` job of this file?

Reply:
Android production job outputs AAB file that we would need to convert to the APK to be uploaded. It's unnecessary complication in my opinion and would have to rely on some converter. This way you just build the APK separately and upload it.

Reply:
Got it 👍